test/container/emqx_container_test.exs

```diff
@@ -0,0 +1,48 @@
+defmodule Testcontainers.Container.EmqxContainerTest do
```

Comment:
you aren't actually doing stuff with the container. Do you think we should add a test to verify we can actually use the emqx container for something "useful" ?

Reply:
Good idea. I will try and come up with something.

Reply:
rebase and fix conflicts. and i can merge it with and you can add test later in a second PR, if you agree